src/features/embalm/stepContent/hooks/useLoadArchaeologists.ts

```diff
@@ -37,8 +37,21 @@ export function useLoadArchaeologists() {
           args: [addresses],
         });
 
+        const stats = await readContract({
+          addressOrName: networkConfig.diamondDeployAddress,
+          contractInterface: ViewStateFacet.abi,
+          functionName: 'getArchaeologistsStatistics',
+          args: [addresses],
+        });
+
         const newArchaeologists = profiles.map((p, i) => ({
```

Comment:
So I'm certainly not trying to dictate coding standards. and I think this is certainly okay. But I generally have a hard and fast rule of no abbreviations of any kind. No need to comment here, just wanted to open the discussion to coding standards ...

Reply:
I think I agree with you here. The `p` should be `profile`. I usually try to do that but this one slipped by me I guess.

Reply:
generally agree with this sentiment. there's contexts where abbreviations make sense, but probably more the exception than the rule